apps/docs/src/examples/extras/trackball-controls/Scene.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import { T } from '@threlte/core'
+  import { TrackballControls } from '@threlte/extras'
+  import { BoxGeometry, MeshStandardMaterial } from 'three'
+</script>
+
+<T.PerspectiveCamera
+  makeDefault
+  position={[10, 5, 10]}
+  lookAt.y={0.5}
+>
+  <TrackballControls />
+</T.PerspectiveCamera>
+
+<T.DirectionalLight
+  position.y={10}
+  position.z={10}
+/>
+<T.AmbientLight intensity={0.3} />
+
+<T.GridHelper args={[10, 10]} />
+
+<T.Mesh
+  position.y={1}
+  geometry={new BoxGeometry(2, 2, 2)}
+  material={new MeshStandardMaterial()}
+/>
```

Comment:
done! I agree that Gizmo is even more valuable when using TrackballControls than when using OrbitControls because it is much easier to get disoriented. However it does highlight that Gizmo could probably be even more helpful if it set the camera up vector as well, at least when using the TrackballControl. I can handle that as a separate PR later.

Reply:
Great, and indeed its a good point the behavior is surprising with `<TrackballControlls>`.

Notes:
- [ ] Let's add some use the `<Gizmo>`'s padding props so it doesn't run up against the edge of the frame; if it takes too much space then you can make it smaller.
- [ ] `<OrbitControls>` deserves a `<Gizmo>` as well!

Reply:
Separate PR regarding `<Gizmo>` up vector handling seems like a good idea to me, but maybe discuss with @grischaerbe before submitting it. Note he is on vacation this week.

Reply:
done!